tests/js/spec/views/stream.spec.jsx

```diff
@@ -19,23 +19,33 @@ const DEFAULT_LINKS_HEADER =
 
 describe('Stream', function() {
   let sandbox;
-  let stubbedApiRequest;
   let context;
   let wrapper;
+  let props;
 
   beforeEach(function() {
     sandbox = sinon.sandbox.create();
-
-    stubbedApiRequest = sandbox.stub(Client.prototype, 'request', (url, options) => {
-      if (
-        url === 'http://127.0.0.1/api/0/projects/sentry/ludic-science/searches/' &&
-        options.method === 'GET'
-      ) {
-        options.success &&
-          options.success([{id: '789', query: 'is:unresolved', name: 'test'}]);
-      }
-      options.complete && options.complete();
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: '/projects/123/456/issues/',
+      body: [TestStubs.Group()],
+      headers: {
+        Link: DEFAULT_LINKS_HEADER,
+      },
+    });
+    MockApiClient.addMockResponse({
+      url: '/projects/123/456/searches/',
+      body: [{id: '789', query: 'is:unresolved', name: 'test'}],
+    });
+    MockApiClient.addMockResponse({
+      url: '/projects/123/456/tags/',
+      body: TestStubs.Tags(),
     });
+    MockApiClient.addMockResponse({
+      url: '/projects/123/456/processingissues/',
+      method: 'GET',
+    });
+    sandbox.stub(browserHistory, 'push');
```

Comment:
if any of these are only used in a single test, it _may_ read better to set them there, but if not nbd

Reply:
I think it's used more broadly, this component changes url to match the query frequently